tactics/hints.ml

```diff
@@ -675,10 +674,10 @@ struct
     let idv = id, { v with db = db.hintdb_name } in
       match gr with
       | None ->
-          let is_present (_, (_, v')) = KerName.equal v.code.uid v'.code.uid in
+          let is_present (_, v') = KerName.equal v.code.uid v'.code.uid in
           if not (List.exists is_present db.hintdb_nopat) then
             (* FIXME *)
```

Comment:
Is this FIXME about what this PR does or something else?

Reply:
I think that it's about the O(n) check in the list, so unrelated.